src/test/HuffDeployer.t.sol

```diff
@@ -229,4 +229,41 @@ contract HuffDeployerTest is Test {
         runTestConstructorCaller(address(0));
         runTestConstructorCaller(address(uint160(0x1000)));
     }
+
+
+    function getBytecode(address contractAddress) view internal returns (bytes memory bytecode) {
+        uint256 codeSize;
+        assembly {
+            codeSize := extcodesize(contractAddress)
+        }
+        bytecode = new bytes(codeSize);
+        assembly {
+            extcodecopy(contractAddress, add(bytecode, 0x20), 0, codeSize)
+        }
+    }
+
+    /// @dev test that compilation is different with new evm versions
+    function testSettingEVMVersion() public {
+        /// expected bytecode for EVM version "paris"
+        bytes memory expectedParis = hex"60028060093d393df36000";
+        HuffConfig config = HuffDeployer.config().with_evm_version("paris");
+        address withParis = config.deploy("test/contracts/EVMVersionCheck");
+
+        bytes memory parisBytecode = getBytecode(withParis);
```

Comment:
You could use directly `withParis.code` and delete `getBytecode` function  if I am not mistaken.